mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -2017,7 +2017,8 @@
 									"    pm.response.to.be.json; ",
 									"});",
 									"var jsonData = pm.response.json();",
-									"pm.environment.set(\"custom-titlename-for-resource-duplicate\", jsonData.data.attributes.name);"
+									"pm.environment.set(\"custom-titlename-for-resource-duplicate\", jsonData.data.attributes.name);",
+									"pm.environment.set(\"custom-titleid-for-resource-2\", jsonData.data.id);"
```

Comment:
Can we have the same naming convention for consistency? when we are saving the titlename we call duplicate where as for title id we say resource 2? Thanks!